app/controllers/lights_controller.rb

```diff
@@ -167,6 +162,27 @@ def set_bulb_from_id
    end
 
    def set_lights
-      @lights = Huey::Bulb.all
+      begin
```

Comment:
Minor comment, `begin` is not required in ruby if the entire `begin-rescue` block starts at the beginning of a  method :)